src/frontend/src/features/ui/components/feedback-widget/index.tsx

```
@@ -0,0 +1,77 @@
+  const channel = process.env.NEXT_PUBLIC_FEEDBACK_WIDGET_CHANNEL;
+  
+  if (!channel || !apiUrl || !widgetPath) return null;
+
+  const title: string = t("feedback_widget.title");
+  const placeholder: string = t("feedback_widget.placeholder");
+  const emailPlaceholder: string = t("feedback_widget.email_placeholder");
+  const submitText: string = t("feedback_widget.submit_text");
+  const successText: string = t("feedback_widget.success_text");
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)